WebRoot/js/zimbraAdmin/common/ZaOverviewPanelController.js

```diff
@@ -1868,13 +1868,10 @@ ZaOverviewPanelController.xformTabTreeListener = function(ev) {
     }
 
     if (ev.refresh) {
-        var currentController = ZaApp.getInstance().getControllerById(viewId);
-        if (!AjxUtil.isEmpty(currentController)) {
-            var currentObject = currentController._currentObject;
-            if (currentObject && currentObject.refresh) {
-                currentObject.refresh(false, true);
-                currentView.setObject(currentObject);
-            }
+        var currentObject = currentView._localXForm.getInstance();
```

Comment:
should we check if currentView._localXForm is not empty using !AjxUtil.isEmpty(currentView._localXForm)
as used on line 1877?

Reply:
`currentView._localXForm` is used in the code without the check You propose, I don't think it is necessary.